### ISSUE TITLE
Additional code comment on separate line?

### ISSUE THREAD

**Issue by stevenmusumeche** [mizdra/eslint-interactive#285]
When using the option "Leave a code comment with your reason for fixing", the comment is placed on the same line as the eslint-disable-next-line comment:

```
            <FadeInUp amount={4}>
              {/* eslint-disable-next-line @typescript-eslint/no-explicit-any -- FIXME http://bit.ly/some-url-here */}
              <Icon icon={MaterialCheckCircle as any} color="$gray40" />
            </FadeInUp>
```

This leads to these lines getting really long and I was wondering if you would consider allowing the comment to be added on the previous line?

```
            <FadeInUp amount={4}>
              {/* FIXME http://bit.ly/some-url-here /*}
              {/* eslint-disable-next-line @typescript-eslint/no-explicit-any */}
              <Icon icon={MaterialCheckCircle as any} color="$gray40" />
            </FadeInUp>
```

I'm happy to submit a PR for this if you can point me in the right direction.

**Comment by mizdra** [comment]
@stevenmusumeche That sounds good! Welcome to submitting the PR.

Basically, it would be easy to implement by mimicking the implementation of the function that allows a description to be added to a comment. Currently eslint-interactive uses the `promptToInputDescription` function to ask the user for a description of the comment. Similarly, it would be nice to implement a `promptToInputDescriptionPosition` function that asks the user where to insert the comment. The obtained answer should be passed to `Core#disablePerLine` or `Core#disablePerFile`.


Inserting a comment is performed on the following line:

https://github.com/mizdra/eslint-interactive/blob/ffb557fae2ee94164167e3ccc1e5a23a00e9c26f/src/plugin/fix/disable-per-line.ts#L55-L61

https://github.com/mizdra/eslint-interactive/blob/ffb557fae2ee94164167e3ccc1e5a23a00e9c26f/src/plugin/fix/disable-per-file.ts#L28-L44

Perhaps the implementation of `toCommentText` needs to be changed as follows.

```diff
-export function toCommentText({ type, scope, ruleIds, description }: Omit<DisableComment, 'range'>): string {
+export function toCommentText({ type, scope, ruleIds, description, descriptionPosition }: Omit<DisableComment, 'range'>): string {
   const header = scope === 'next-line' ? 'eslint-disable-next-line' : 'eslint-disable';
   const ruleList = unique(ruleIds).join(', ');
   if (type === 'Line') {
     if (description === undefined) {
       return `// ${header} ${ruleList}`;
     } else {
-      return `// ${header} ${ruleList} -- ${description}`;
+      if (descriptionPosition === 'separate') {
+        return `// ${description}\n// ${header} ${ruleList}`;
+      } else {
+        return `// ${header} ${ruleList} -- ${description}`;
+      }
     }
   } else {
     if (description === undefined) {
       return `/* ${header} ${ruleList} */`;
     } else {
-      return `/* ${header} ${ruleList} -- ${description} */`;
+      if (descriptionPosition === 'separate') {
+        return `/* ${description} */\n/* ${header} ${ruleList} */`;
+      } else {
+        return `/* ${header} ${ruleList} -- ${description} */`;
+      }
     }
   }
 }
```

**Comment by stevenmusumeche** [comment]
@mizdra - how can I run the CLI locally to test my changes?

**Comment by stevenmusumeche** [comment]
Thanks! I've just opened a PR @mizdra

**Comment by mizdra** [comment]
This feature is shipped by [v10.8.0](https://github.com/mizdra/eslint-interactive/releases/tag/v10.8.0)!
